content/en/docs/concepts/workloads/controllers/daemonset.md

```diff
@@ -168,15 +165,15 @@ Although Daemon Pods respect
-| `node.kubernetes.io/not-ready`           | NoExecute  | `TaintBasedEvictions`                                        | 1.8+    | when `TaintBasedEvictions`  is enabled,they will not be evicted when there are node problems such as a network partition. |
-| `node.kubernetes.io/unreachable`         | NoExecute  | `TaintBasedEvictions`                                        | 1.8+    | when `TaintBasedEvictions`  is enabled,they will not be evicted when there are node problems such as a network partition. |
-| `node.kubernetes.io/disk-pressure`       | NoSchedule | `TaintNodesByCondition`                                      | 1.8+    |                                                              |
-| `node.kubernetes.io/memory-pressure`     | NoSchedule | `TaintNodesByCondition`                                      | 1.8+    |                                                              |
-| `node.kubernetes.io/unschedulable`       | NoSchedule | `ScheduleDaemonSetPods`, `TaintNodesByCondition`             | 1.11+   | When ` ScheduleDaemonSetPods` is enabled, ` TaintNodesByCondition` is necessary to make sure DaemonSet pods tolerate unschedulable attributes by default scheduler. |
-| `node.kubernetes.io/network-unavailable` | NoSchedule | `ScheduleDaemonSetPods`, `TaintNodesByCondition`, hostnework | 1.11+   | When ` ScheduleDaemonSetPods` is enabled, ` TaintNodesByCondition` is necessary to make sure DaemonSet pods, who uses host network, tolerate network-unavailable attributes by default scheduler. |
-| `node.kubernetes.io/out-of-disk`         | NoSchedule | `ExperimentalCriticalPodAnnotation` (critical pod only), `TaintNodesByCondition` | 1.8+    |                                                              |
+| `node.kubernetes.io/not-ready`           | NoExecute  | `TaintBasedEvictions` - Alpha                                       | 1.8+    | when `TaintBasedEvictions` is enabled, they will not be evicted when there are node problems such as a network partition. |
+| `node.kubernetes.io/unreachable`         | NoExecute  | `TaintBasedEvictions` - Alpha                                        | 1.8+    | when `TaintBasedEvictions` is enabled, they will not be evicted when there are node problems such as a network partition. |
+| `node.kubernetes.io/disk-pressure`       | NoSchedule | `TaintNodesByCondition` - Beta                                      | 1.12+    |                                                              |
+| `node.kubernetes.io/memory-pressure`     | NoSchedule | `TaintNodesByCondition` - Beta                                      | 1.12+    |                                                              |
+| `node.kubernetes.io/unschedulable`       | NoSchedule | `ScheduleDaemonSetPods` - Alpha</br> `TaintNodesByCondition` - Beta</br>             | 1.11+</br> 1.12+</br>   | When ` ScheduleDaemonSetPods` is enabled, DaemonSet pods tolerate unschedulable attributes by default scheduler. |
```

Comment:
I am not so sure about these line breaks in the table. Since different cells have different amount of text, the lines in two columns of the same row are not perfectly aligned. Some users may not realize that the versions of K8s correspond to the features.
On another note, I am not sure if we should put Beta features in this table. Beta features are enabled by default and may not be needed here.

Reply:
There are 3 flags here described as alpha features. But 2 of them (TaintNodesByCondition and ScheduleDaemonSetPods) are going to be beta in 1.12, so I changed the column name from "Alpha" to "Alpha/Beta", and split into two lines using `</br>` as well as in "Version" column.

It might look a little tricky.. So if beta features are not needed to mention here, things are easier. @bsalamat Are you ok with removing beta feature flags in this table?

Reply:
I think it would be better to remove the beta features as they are enabled by default. Those wondering what features where in alpha in older releases could refer to 1.11 documentation.

Reply:
Done.